code/modules/clothing/spacesuits/alien_suits.dm

```diff
@@ -143,30 +143,48 @@
 	desc = "An almost organic looking nonhuman pressure suit."
 
 /obj/item/clothing/under/vox
-	has_sensor = FALSE
-	icon = 'icons/obj/clothing/species/vox/uniforms.dmi'
+	icon = 'icons/mob/clothing/species/vox/under/misc.dmi'
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/under/misc.dmi'
 		)
 
 /obj/item/clothing/under/vox/vox_casual
-	name = "alien clothing"
+	name = "Vox jumpsuit"
 	desc = "This doesn't look very comfortable."
-	icon_state = "vox-casual-1"
-	item_color = "vox-casual-1"
-	item_state = "vox-casual-1"
+	icon_state = "vox-jumpsuit_s"
+	item_color = "vox-jumpsuit"
+	item_state = "vox-jumpsuit"
 	body_parts_covered = LEGS
 
-/obj/item/clothing/under/vox/vox_robes
-	name = "alien robes"
-	desc = "Weird and flowing!"
-	icon_state = "vox-casual-2"
-	item_color = "vox-casual-2"
-	item_state = "vox-casual-2"
+/obj/item/clothing/suit/hooded/vox_robes
+	name = "Vox robes"
+	desc = "Weird and flowing robes!"
+	icon = 'icons/mob/clothing/species/vox/suit.dmi'
+	icon_state = "vox-robes"
+	item_color = "vox-robes"
+	item_state = "vox-robes"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS
+	hoodtype = /obj/item/clothing/head/hooded/vox_robe_hood
+	species_restricted = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+//Vox Robes Hood
+/obj/item/clothing/head/hooded/vox_robe_hood
+	name = "Vox hood"
+	desc = "It's hood that covers the head, made of a weird and flowing material."
```

Comment:
See if someone from the lore team has any description suggestions, its a good place for pieces of the species lore.

Also, which hood doesnt cover the head??

Reply:
Loreful descriptions have been added, with thanks to Octus for assisting with lore.